src/wmc.rs

```diff
@@ -0,0 +1,566 @@
+// New Java switches are not correctly recognised by RCA, which returns some error nodes.
```

Comment:
It'd be nice to have here and elsewhere in this file some urls linking to some useful documentation about wmc and the way to compute it.

Reply:
I reported the same paper cited in the PR also in a comment